scripts/pylib/twister/twisterlib/testinstance.py

```diff
@@ -204,9 +209,14 @@ def setup_handler(self, env):
     # Global testsuite parameters
     def check_runnable(self, enable_slow=False, filter='buildable', fixtures=[], hardware_map=None):
 
-        # running on simulators is currently not supported on Windows
-        if os.name == 'nt' and self.platform.simulation != 'na':
-            return False
+        if os.name == 'nt':
+            # running on simulators is currently supported only for QEMU on Windows
+            if self.platform.simulation not in ('na', 'qemu'):
+                return False
+
+            # check presence of QEMU on Windows
+            if 'QEMU_BIN_PATH' not in os.environ:
```

Comment:
This line runs even if you're not simulating on QEMU, so under windows on-device testing is broken if you don't have QEMU set up.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/70267